DGC.Types.schema.json

```diff
@@ -166,12 +163,12 @@
         "df": {
           "description": "ISO 8601 Date: Certificate Valid From",
           "type": "string",
-          "format": "date-time"
```

Comment:
This has to be date-time: was a question I had to the Semantic SG who confirmed it was not date-only